### DISCUSSION TITLE
Apply Github Developer Student Pack while not physically on campus

### DISCUSSION THREAD

**Discussion by CheeMinHao** [community/community#147877] [GitHub Education]
### Hi everyone,

I'm currently on break and hence have went back to my hometown and I would like to apply for the pack asap. However due to the location tracking requirement I can only wait until I get back to the country where my campus is located to apply for it. May I know if there's any other alternative for this or do I really have to wait until im back (which is like, in 2 months)?

**Comment by davevad93** [comment]
Hi @CheeMinHao ,

Unfortunately, you have to be close to your campus in order to successfully apply for the `Student Developer Pack`, so in your case you'll have to wait until you return from the school break.
